Lib/inspect.py

```diff
@@ -2126,11 +2131,29 @@ def _signature_get_partial(wrapped_sig, partial, extra_args=()):
     return wrapped_sig.replace(parameters=new_params.values())
 
 
+def _multisignature_get_partial(wrapped_sig, partial, extra_args=()):
+    last_exc = None
```

Comment:
Unless the case of the list being empty is handled elsewhere, this could result in a `raise None`. Maybe that None should be something like `Exception("Empty MultiSignature list")` ?

Reply:
It is checked in the constructor. I think even about removing this assignment, NameError is clearer indication if something goes wrong.

Reply:
I agree.